ccmlib/scylla_repository.py

```diff
@@ -90,13 +94,6 @@ def download_version(version, url=None, verbose=False, target_dir=None):
     except tarfile.ReadError as e:
         raise ArgumentError("Unable to uncompress downloaded file: %s" % str(e))
     except CCMError as e:
-        if target_dir:
-            # wipe out the directory if anything goes wrong.
-            try:
-                rmdirs(target_dir)
-                print_("Deleted %s due to error" % target_dir)
-            except:
-                raise CCMError("Downloading/extracting scylla version %s failed. Attempted to delete %s but failed. This will need to be manually deleted" % (version, target_dir))
```

Comment:
why?

Reply:
this code isn't needed anymore, since we work with temp directory to begin with.
and it was only masking the error being raised

Reply:
@fruch So you can remove the whole `except CCMError` part, no?

Reply:
yep, you got a point there, removing